concrete/blocks/express_form/controller.php

```diff
@@ -418,12 +420,25 @@ public function action_update_control()
 
     public function save($data)
     {
+        $entityManager = $this->app->make(EntityManagerInterface::class);
+
         if (isset($data['exFormID']) && $data['exFormID'] != '') {
+            $this->exFormID=$data['exFormID'];
```

Comment:
If the express entity form block added from scrapbook is modified then the associated entity is duplicated with updating duplicate block data.